src/main/scala/it/pagopa/interop/catalogprocess/common/readmodel/ReadModelQueries.scala

```diff
@@ -73,10 +73,13 @@ object ReadModelQueries {
     producersIds: Seq[String],
     states: Seq[EServiceDescriptorState],
     offset: Int,
-    limit: Int
+    limit: Int,
+    exactMatchOnName: Boolean = false
   )(readModel: ReadModelService)(implicit ec: ExecutionContext): Future[PaginatedResult[CatalogItem]] = {
 
-    val query = listEServicesFilters(name, eServicesIds, producersIds, states)
+    val query =
+      if (exactMatchOnName) listEServicesFilters(name, eServicesIds, producersIds, states, exactMatchOnName = true)
+      else listEServicesFilters(name, eServicesIds, producersIds, states)
```

Comment:
this is equivalent to
```suggestion
    val query = listEServicesFilters(name, eServicesIds, producersIds, states, exactMatchOnName)
```